microsetta_public_api/models/_taxonomy.py

```diff
@@ -107,13 +97,14 @@ def __init__(self, table: biom.Table, features: pd.DataFrame,
             DataFrame relating an observation to a Taxon
         variances : biom.Table, optional
             Variation information about a taxon within a label.
+        rank_level : int
+            The taxonomic level to compute ranked data over
```

Comment:
I think 'level' could be a little ambiguous, i.e., it is not immediately clear (to me) what that precisely corresponds to. It seems like this is depth in the tree? Maybe something `The taxonomic level (depth) ...` would be good?

Reply:
Sounds good, will expand